tools/github/hooks/prerun.sh

```diff
@@ -9,25 +9,40 @@ echo "GITHUB_RUN_ID=$GITHUB_RUN_ID"
 echo "GITHUB_RUN_ATTEMPT=$GITHUB_RUN_ATTEMPT"
 echo "GITHUB_JOB=$GITHUB_JOB"
 echo "GITHUB_REPOSITORY=$GITHUB_REPOSITORY"
+echo "GITHUB_BASE_REF=$GITHUB_BASE_REF"
+echo "GITHUB_HEAD_REF=$GITHUB_HEAD_REF"
+echo "GITHUB_REF=$GITHUB_REF"
+echo "GITHUB_REF_TYPE=$GITHUB_REF_TYPE"
```

Comment:
Do we need to log these? These are shown to the user so might want to keep it clean.

Reply:
These would help in debugging workflows in the future if they fail.